solutions/security/dashboards/endpoint-detection-response-dashboard.md

```diff
@@ -0,0 +1,68 @@
+---
+applies_to:
+  stack: ga 9.2
+  serverless:
+    security: ga
+products:
+  - id: security
+  - id: cloud-serverless
+---
+
+# Endpoint Detection and Response dashboard
+
+The Endpoint Detection and Response (EDR) dashboard provides visualizations to help you monitor and analyze detection and prevention activity on endpoints running {{elastic-defend}}. Use this dashboard to view total detection, prevention, and ransomware alerts, as well as open alerts by severity, MITRE technique, and operating system. The dashboard also highlights the top 10 most infected endpoints and the top 10 most impacted users per endpoint.
+
+With these insights, security analysts and administrators can quickly identify high-risk endpoints or potentially compromised users, evaluate detection rule performance, understand environment coverage, and pivot directly into the relevant alert details to begin investigations.
+
+To access the dashboard:
+1. Find **Dashboards** using the navigation menu or the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md).
+2. In the **Custom Dashboards** section, search for "Endpoint Detection and Response".
+
+:::{image} /solutions/images/security-endpoint-detection-response-dashboard.png
+:alt: Endpoint Detection and Response dashboard
+:::
+
+::::{admonition} Requirements
+To access this dashboard and its data, you must have:
+
+* At least `Read` [{{kib}} privileges](/deploy-manage/users-roles/cluster-or-deployment-auth/kibana-role-management.md) for both the **Analytics > Dashboard** and **Security > Security** {{kib}} features.
+* At least `read` [index privileges](/deploy-manage/users-roles/cluster-or-deployment-auth/kibana-role-management.md#adding_index_privileges) for the `.kibana-event-log-*` index.
```

Comment:
This can be removed

```suggestion
* At least `read` [index privileges](/deploy-manage/users-roles/cluster-or-deployment-auth/kibana-role-management.md#adding_index_privileges) for the `.kibana-event-log-*` index.
```